app/assets/javascripts/discourse/ember/resolver.js.es6

```diff
@@ -188,7 +188,8 @@ export default Ember.DefaultResolver.extend({
       const dashed = decamelized.replace(/_/g, '-');
       return Ember.TEMPLATES[decamelized] ||
              Ember.TEMPLATES[dashed] ||
-             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')];
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')] ||
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/templates/')];
```

Comment:
I haven't grokked the plugin assets system 100% yet, but it seems like the difference may be that plugin assets are stored under `javascripts/admin/templates/`, whereas discourse core assets are stored under just `admin/templates/`

So, for plugin assets:
tries this: `javascripts/admin/chat` (no good because `javascripts/admin/chat` isn't a thing)
and this: `admin/templates/chat` (no good because we'd need `javascripts` at the root there)

However, for normal assets:
tries this: `javascripts/admin/groups` (no good, because it's not in the javascripts/ folder)
and this: `admin/templates/groups` (works!)

I think this is unique to the admin assets, because they're a little weird, all the others will come in with dot separators (eg `common.header`), while the admin ones are camelized (eg `adminGroups`).

There's even a few hacks in the resolver already around admin routes, such as `resolver.js.es6#179`:
```
    if (decamelized === "javascripts/admin") {
      return Ember.TEMPLATES['admin/templates/admin'];
    }
```